data/slowcheetah.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "Himadri Bhattacharya",
+	"type": "personal",
+	"bio": "I am a full stack developer",
+	"avatar": "https://github.com/slowcheetah07.png"
```

Comment:
Also, we no longer need the `avatar` key, it will automatically pick your profile from GitHub. So please remove the `avatar` key from your `json`.

```suggestion
```